banhmi/src/controllers/auth.ts

```diff
@@ -0,0 +1,102 @@
+import { prisma } from "@/db";
```

Comment:
I think we should run prettier on all files to follow the same rules (e.g. 2 spaces for tab, include semicolon, consistent import order, etc.)